### DISCUSSION TITLE
What age is the game suitable for?

### DISCUSSION THREAD

**Discussion by MiguelRS-13** [FunkinCrew/Funkin#1511] [Q&A]
Hi everyone, I have a question about the content of the game “Friday Night Funkin”. While playing, I realized that there were signs that the game is for people over 18.
Examples: Phrase that appears at the beginning of the game and Senpai's speech before the second song.
Knowing this, I stopped playing (Note: I am 13 years old). Then I discovered that some people of the same age (or even minors) followed and played the game.
Well, my question is (or questions): is it possible for the game to change in that direction? Is it possible for these things to be removed? Or is it even recommended for people over 18? I definitely love the game and really wanted to keep playing, but unfortunately I can't because of that.

**Comment by TentaRJ** [comment]
### It's implied that the game should be focused on the audience of people that are more suited with **Newgrounds** due to how many references there are to the site. I can see where you are coming from though.

**Comment by isakube** [comment]
a 13yo sounding like a concerned parent? you see something new every day

Funkin' is listed on Newgrounds as rated T so that's like 13+. pretty inaccurate though imo

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa4** [comment]
cap john straight up says cunt so I think it's around 16+ but classified as 13+

**Comment by Lasercar** [comment]
Wait until the game is on steam and find out
